tools/docker-network/tests/committeerotation_test.go

```diff
@@ -0,0 +1,118 @@
+package tests
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/iotaledger/hive.go/ierrors"
+	iotago "github.com/iotaledger/iota.go/v4"
+	"github.com/stretchr/testify/require"
+)
+
+var ValidatorContainerNames = map[string]string{
+	"V1": "docker-network-inx-validator-1-1",
+	"V2": "docker-network-inx-validator-2-1",
+	"V3": "docker-network-inx-validator-3-1",
+	"V4": "docker-network-inx-validator-4-1",
+}
+
+var ClientURLs = map[string]string{
+	"V1":    "http://localhost:8050",
+	"V2":    "http://localhost:8060",
+	"V3":    "http://localhost:8070",
+	"V4":    "http://localhost:8040",
+	"node5": "http://localhost:8090",
+}
```

Comment:
Instead of having these separate we could create a struct `Node` that holds all this information. Again, this could be similar (but simplified) as in the testsuite. Then we could have some convenience functions that gives you the name of a node's validator container which you could then hand to the `StopContainer` method. 

Or another thing could be `d.AccountsFromNodes(d.Nodes("V1","V2"))` which we could then hand into `AssertCommitteeSelection`.

This should be a bit cleaner and easier to use.